features/works/work_related.feature

```diff
@@ -284,13 +284,14 @@ Scenario: Listing external works as inspirations
   When I view my related works
   Then I should see "From N/A to English"
   # inactive URL should give a helpful message (AO3-1783)
+  # unreachable URL should give a more helpful message (A03-3536)
```

Comment:
I don't think we need to keep either of these messages anymore - git blame will tell us which tickets this test was designed to verify